README.md

```diff
@@ -1,20 +1,18 @@
-GnuCash Web
-===========
+# GnuCash Web
```

Comment:
Why use the hashtag notation? I like the underlines, as it is also readable in plaintext.